realtime/redisPublisher.js

```diff
@@ -142,7 +144,11 @@ function publishSample(sampleInst, subjectModel, event, aspectModel) {
     const asp = responses[0];
     const sub = responses[1];
 
-    sample.aspect = asp.get ? asp.get() : asp;
+    let aspect = asp.get ? asp.get() : asp;
+    aspect = redisStore.removeNulls(aspect,
+      constants.fieldsToStringify.aspect);
+    delete aspect.writers;
+    sample = cleanAddAspectToSample(sample, aspect);
```

Comment:
Also, samples from DB already have aspects. Probably will have to work around the samples from DB